routes/web.php

```diff
@@ -25,7 +25,7 @@
 });
 
 Route::prefix('pet')->middleware('auth')->group(function () {
-    Route::get('/',[PetController::class, 'index'])->name('indexPet');
+    Route::get('index',[PetController::class, 'index'])->name('indexPet');
```

Comment:
/pet/index to bylo na stronach 10 lat temu wariacie, zostaw jak bylo

Reply:
poprawione